src/pair.py

```diff
@@ -459,3 +459,64 @@ def __init__(self, nlist, default_r_cut=None, mode='none'):
             ),
         )
         self._add_typeparam(params)
+
+
+class ExpandedYukawa(pair.Pair):
+    r"""Expanded Yukawa pair potential.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list.
+        r_cut (float):cDefault cutoff radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `ExpandedYukawa` is a shifted Yukawa potential. The discontinuity is shifted
+    based on a :math:`\Delta` that is the minimum interaction distance between
+    particles.
+
+    The `ExpandedYukawa` potential is:
+
+    .. math::
+
+        U(r) = \varepsilon \frac{\exp[-\kappa(r - \Delta)]}{r - \Delta}
```

Comment:
Are there any safeguards for when `r = \Delta`? Should this case be tested to insure behavior is as-intended ?

Reply:
There are no safeguards for that case. When r = \Delta, that means that the particles are touching and the potential and force should go to infinity. Looking specifically at the expanded mie hoomd source code, there is no specific error message for when two particles touch/overlap beyond the one detailing how particles are no longer inside of the box. In this case, we are handling the error in the same way hoomd does.